test

```diff
@@ -24,10 +24,6 @@ trap "rm -rf _testbin/" EXIT
 pkgs=$(go list $PKG | grep -v /vendor/)
 src=$(find . -name '*.go' -not -path "./vendor/*")
 
-echo "Building tests..."
-go test -mod=vendor -i "$@" $pkgs
-go install -mod=vendor $pkgs
```

Comment:
Should we keep this? Otherwise we need to also remove the `Running commands...` section

Reply:
yes bringing it back